src/main/java/com/openlattice/authorization/HazelcastSecurableObjectResolveTypeService.java

```diff
@@ -59,4 +60,17 @@ public SecurableObjectType getSecurableObjectType( AclKey aclKey ) {
         return securableObjectTypes
                 .keySet( Predicates.equal( SecurableObjectTypeMapstore.SECURABLE_OBJECT_TYPE_INDEX, type ) );
     }
+
+    @Override
+    public Set<AclKey> getOrganizationExternalDatabaseAclKeys( List<Acl> acls ) {
+        return securableObjectTypes.keySet(
```

Comment:
I think this is missing to match with keys of the `acls`

Reply:
Oh yes! Will fix.